src/wp-includes/ms-network.php

```diff
@@ -110,9 +109,14 @@ function clean_network_cache( $ids ) {
  * @param array $networks Array of network row objects.
  */
 function update_network_cache( $networks ) {
+	if ( ! $networks ) {
```

Comment:
why do you need this check? would calling `wp_cache_add_multiple` with an empty array cause problems?